### DISCUSSION TITLE
New components in the pipeline

### DISCUSSION THREAD

**Discussion by nouman91** [tailwindlabs/headlessui#1032] [Ideas]
Hi,

Thank you so much for this amazing library. I have used it in a couple of applications, worked really well. However, this time around I am working on a large-scale platform application for that I need a lot of components. 

Can you share the future roadmap of what/when are you planning to release new components? 

Much appreaciated!

**Comment by andyjessop** [comment]
I'm also interested in this. Clearly the idea behind this project is to create only the building blocks, so that you can compose your own components. However, this does leave some things unanswered, like what sort of components does the team believe can be composed from current crop, and what components _are_ planned (or at least envisaged by the team)?

It would also be good to see some examples of components that exist in other UI libraries but that are not ever intended to be built for HeadlessUI.

Love the library, by the way - keep up the good work!
